packages/flutter/test/material/outline_button_test.dart

```diff
@@ -44,78 +44,98 @@ void main() {
   });
 
 
-  testWidgets('Outline shape and border overrides', (WidgetTester tester) async {
-    debugDisableShadows = false;
-    const Color fillColor = Color(0xFF00FF00);
-    const Color borderColor = Color(0xFFFF0000);
-    const Color highlightedBorderColor = Color(0xFF0000FF);
+  testWidgets('OutlineButton shape and border component overrides', (WidgetTester tester) async {
+    const Color fillColor = const Color(0xFF00FF00);
+    const Color borderColor = const Color(0xFFFF0000);
+    const Color highlightedBorderColor = const Color(0xFF0000FF);
+    const Color disabledBorderColor = const Color(0xFFFF00FF);
     const double borderWidth = 4.0;
 
-    await tester.pumpWidget(
-      new Directionality(
+    Widget buildFrame(Key key, VoidCallback onPressed) {
```

Comment:
The key parameter doesn't seem necessary, wouldn't `find.byType(OutlineButton)` be sufficient?

Also: even if this method only has one parameter, the calls will be a little easier to read with named parameters:
```
buildFrame(onPressed: null)
```

Reply:
key removed, changed to named params